func.cpp

```diff
@@ -0,0 +1,126 @@
+#include <stdio.h>
+#include <locale.h>
+#include <stdlib.h>
+#include <ctype.h>
+#define N 32
+void menu()
+int vvod_mas(int **a)
+void vivod_mas(int n, int m, int **a)
+int 1zadanie(int n, int m, int **a)
+int 2zadanie()
+
+int main()
+{
+    setlocale(LC_ALL, "rus");
+    int menu();
+
```

Comment:
clean up extra enters

Reply:
Now the console will display Russian letters

Reply:
If we don't use Russuin at all, then 14 line sholud be deleted